docs/csharp/language-reference/builtin-types/enum.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Enumeration types - C# reference"
+description: "Learn about C# enumeration types that represent a set of named constants and how to define them with the `enum` keyword"
+ms.date: 09/12/2019
+f1_keywords:
+  - "enum"
+  - "enum_CSharpKeyword"
+helpviewer_keywords:
+  - "enum keyword [C#]"
+  - "enum type [C#]"
+  - "enumeration type [C#]"
+  - "bit flags [C#]"
+ms.assetid: bbeb9a0f-e9b3-41ab-b0a6-c41b1a08974c
+---
+# Enumeration types (C# reference)
+
+An enumeration type (or enum type) is a value type that represents a set of named integral constants. To define an enumeration type, use the `enum` keyword and specify the names of *enum members*:
+
+```csharp
+enum Season
+{
+    Spring,
+    Summer,
+    Autumn,
+    Winter
+}
+```
```

Comment:
Why not reference these lines from the snippets repo ?

Reply:
I think such little examples easier to maintain inline as it is

Reply:
We've noticed that, as a team, we've adopted slightly different strategies for managing snippets, full programs, tutorials and examples. We're starting discussions to see what we can share, and if there is a common set of practices we should use. 

We'll update the contributing articles once we've had those discussions. In the meantime, this is fine.